metadata.json

```diff
@@ -15,6 +15,10 @@
     "alias"
   ],
   "dependencies": [
+     {
```

Comment:
yea, and it doesn't do anything but lay down the functions. It will be required for 7.x